docs/jwst/outlier_detection/arguments.rst

```diff
@@ -1,80 +1,80 @@
 .. _outlier_detection_step_args:
 
+For more details about step arguments (including datatypes, possible values
+and defaults) see :py:obj:`jwst.outlier_detection.OutlierDetectionStep.spec`.
+
 Step Arguments for Non-IFU data
 ===============================
 The `outlier_detection` step for non-IFU data has the following optional arguments
 that control the behavior of the processing:
 
-``--weight_type`` (string, default='exptime')
-  The type of data weighting to use during resampling;
-  options are 'exptime', 'error', and 'None'.
+``--weight_type``
+  The type of data weighting to use during resampling.
 
-``--pixfrac`` (float, default=1.0)
-  The pixel fraction used during resampling;
-  valid values go from 0.0 to 1.0.
```

Comment:
I think this statement could be retained.

Reply:
I re-added it. I removed it because the spec doesn't have a min or max value for this parameter.
https://github.com/spacetelescope/jwst/blob/619cb3bbf6e6dd122ad807cf73705c5c81c49c06/jwst/outlier_detection/outlier_detection_step.py#L55
unlike in resample:
https://github.com/spacetelescope/jwst/blob/619cb3bbf6e6dd122ad807cf73705c5c81c49c06/jwst/resample/resample_step.py#L43
So with the current outlier detection providing a <0 or >1 pixfrac won't produce a `ValidationError`.

Reply:
Yikes, hadn't noticed that. I think this argues for an update to the spec to include those min/max values, just like resample. Values outside that range really are unphysical and should not be allowed.

Reply:
A problem for another day? https://github.com/spacetelescope/jwst/issues/8610

Reply:
> A problem for another day? #8610

Sure. The lack of min/max for pixfrac apparently hasn't caused any problems (yet), and we want to take some time to consider whether there are any legitimate reasons for any of the others to be different, so let's wait with all of them and keep this as-is for now.